addons/website/models/ir_ui_view.py

```diff
@@ -240,6 +240,26 @@ def _get_top_level_view(self):
         self.ensure_one()
         return self.inherit_id._get_top_level_view() if self.inherit_id else self
 
+    def get_view_hierarchy(self):
+        self.ensure_one()
+        top_level_view = self._get_top_level_view().with_context(active_test=False)
+        sibling_views = top_level_view.search_read([('key', '=', top_level_view.key), ('id', "!=", top_level_view.id)])
+        return {'sibling_views':sibling_views, "hierarchy": top_level_view._build_hierarchy_datastructure()}
+
+    def _build_hierarchy_datastructure(self):
+        inherit_children = []
+        for child in self.inherit_children_ids:
+            inherit_children.append(child._build_hierarchy_datastructure())
+        return {
+            "id": self.id,
```

Comment:
nitpicking: we try to use single quote in this case and use double quote for literal strings